nxc/protocols/winrm.py

```diff
@@ -4,9 +4,12 @@
 import hashlib
 import os
 import requests
+import urllib3
```

Comment:
```suggestion
import urllib3
```

Reply:
[](https://www.bloomberg.com/news/articles/2023-10-13/sec-won-t-ask-court-to-reverse-its-decision-on-spot-bitcoin-etf?utm_source=website&utm_medium=share&utm_campaign=copy)